src/Components/UpdateArticleVotes.jsx

```diff
@@ -0,0 +1,60 @@
+import { getArticle, updateVotes } from "../Utility";
+import { useEffect, useState } from "react";
+import staticlike from "../assets/staticlike.png";
+import dislike from "../assets/dislike.png"
+
+function UpdateArticleVotes({ articleId }) {
+  const [votes, setVotes] = useState(0);
+  const [err, setErr] = useState(null);
+
+  useEffect(() => {
+    getArticle(articleId).then((singleArticle) => {
```

Comment:
do you need to do another API call? you could pass the votes down from the singleArticle component